spec/feature/mission_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe 'Missions List', type: :feature do
+  subject { page }
+  
+  let(:mission) { create(:mission) }
+
+  describe 'Showing missions' do
+    context 'when there is no misson' do
+      before { visit missions_path }
+      
+      it_behaves_like 'a missions list page'
+    end
+
+    context 'when there are missions' do
+      before do
+        mission
+        visit missions_path
+      end
+  
+      it_behaves_like 'a missions list page'
+      it_behaves_like 'a page that shows the missions'
+    end
+  end
+    
+  describe 'Adding missions' do
+    let(:mission) { build(:mission)}
+    
+    before do
+      visit new_mission_path
+      fill_in_form(mission)
+      click_button '建立任務'
+    end
+    
+    it_behaves_like 'a missions list page'
+    it_behaves_like 'a page that shows the missions'
+  end
+
+  describe 'Updating missions' do
+    before do
+      to_be_updated_mission = create(:mission)
+      visit edit_mission_path(to_be_updated_mission)
+      fill_in_form(mission)
+      click_button '修改任務'
+    end
+
+    it_behaves_like 'a missions list page'
+    it_behaves_like 'a page that shows the missions'
+  end
+
+  describe 'Deleting missions' do
+    before do
+      mission
+      visit missions_path
+      find(destroy_link(mission)).click
+    end
```

Comment:
為了明確表達你的意圖，請改用類似以下的寫法：
```suggestion
      let!(:mission) { create(:mission) }

    before do
      visit missions_path
      find(destroy_link(mission)).click
    end
```